src/Microsoft.DotNet.Build.Tasks/Targets/PerfTesting.targets

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <!-- properties for consumptive metric testing -->
+  <PropertyGroup Condition="'$(TestConsumptiveMetrics)' == 'true'">
+    <BaselineDataFile>$(MSBuildProjectDirectory)\CBM.$(MSBuildProjectName).xml</BaselineDataFile>
+    <CollectConsumptiveMetrics>true</CollectConsumptiveMetrics>
+    <CollectPerfEvents>true</CollectPerfEvents>
+  </PropertyGroup>
+
+  <!--
+    Perf tools package versions go here and in the test-runtime-packages.config
+  -->
+  <PropertyGroup>
+    <PerfToolsVersion>0.0.1-prerelease-00018</PerfToolsVersion>
+    <PerfToolsDir Condition="'$(PerfToolsDir)'==''">$(PackagesDir)Microsoft.DotNet.PerfTools.$(PerfToolsVersion)\tools\</PerfToolsDir>
+    <EventTracer>"$(PerfToolsDir)EventTracer.exe"</EventTracer>
+    <!-- by default delete the trace files after consumption -->
+    <DeletePerfDataFile Condition="'$(DeletePerfDataFile)' == ''">true</DeletePerfDataFile>
+    <!-- run tests is serial when collecting perf data so they don't all bleed together -->
+    <SerializeProjects Condition="'$(CollectPerfEvents)' == 'true'">true</SerializeProjects>
+  </PropertyGroup>
+
+  <!-- perf events for consumptive metrics testing -->
+  <PropertyGroup Condition="'$(CollectConsumptiveMetrics)' == 'true'">
+    <ClrMetrics>GCSampledObjectAllocationHigh,GCSampledObjectAllocationLow</ClrMetrics>
```

Comment:
Should this be changed to ClrMetrics Condition="'$(ClrMetrics)' == ''"?  That way individual projects could overload the metric types.